### DISCUSSION TITLE
How to use mindmap

### DISCUSSION THREAD

**Discussion by Mister-Hope** [mermaid-js/mermaid#3763] [Q&A] [answered]
Mermaid seems not bundling mindmap by default in 9.2.0, I don't know how to use it.

It's available at live editor, but not with mermaid.

There is only some pieces which talks about adding digram at  https://mermaid-js.github.io/mermaid/#/newDiagram,

And I don't see how to register it like https://github.com/mermaid-js/mermaid/blob/e876c35ef97d60735b8f1305c1590bf61a6759a8/packages/mermaid/src/diagram-api/diagram-orchestration.ts

**Comment by sidharthv96** [comment, marked as answer]
Hi @Mister-Hope, I've added documentation for adding mindmaps in https://github.com/mermaid-js/mermaid/pull/3810


> Mindmap uses the experimental lazy loading & async rendering features which could change in the future.
>
> 
> ```html
> <script type="module">
>   import mermaid from 'https://unpkg.com/mermaid@9/dist/mermaid.esm.min.mjs';
>   import mindmap from 'https://unpkg.com/@mermaid-js/mermaid-mindmap@9/dist/mermaid-mindmap.esm.min.mjs';
>   await mermaid.registerExternalDiagrams([mindmap]);
> </script>
> ```
> 
> You can also refer the implementation in the live editor [here](https://github.com/mermaid-js/mermaid-live-editor/blob/fcf53c98c25604c90a218104268c339be53035a6/src/lib/util/mermaid.ts) to see how the async loading is done.
>

**Reply by Mister-Hope** [reply]
Hi, 9.4 introduce breaking changes by removing the package from repo, so this is a breaking change, but this is not mentioned, could you edit release note and changelog to mention the change?

Also mindmap and timeline added in recent releases are having problems in 9.4, chould you check https://github.com/mermaid-js/mermaid/issues/4089
